### PR TITLE
WIP(emotion,instui-config,ui-buttons,ui-docs-client,ui-editable,ui-fi…

### PR DESCRIPTION
**Pull request by ke1k0**
…le-drop,ui-flex,ui-modal,ui-tab

Closes: #170

Flex and FlexItem converted to functional component
themeable migrated to emotion

BREAKING CHANGE:
FlexItem no longer accessible as Flex.Item, has to be imported separately and used as FlexItem

### CODE REVIEW

**Review by matyasf — APPROVED** [review]
Looks good, although its hard to review with all these linter changes :/ Cant wait until the global linter change is merged

File: packages/ui-flex/src/Flex/Item/styles.js
Comment (Brailor) [inline packages/ui-flex/src/Flex/Item/styles.js:45]: Empty line

Comment (ke1k0) [reply packages/ui-flex/src/Flex/Item/styles.js]: fixed

**Pull request merged**
